### PR TITLE
feat(client): using task metrics

### PR DESCRIPTION
**Pull request by frascuchon**
With this PR you can cool things like:

```python
from rubrix.metrics.text_classification import f1

results = f1(dataset)
results.visualize() # you must install plotly before
# or give raw data
results.data
```

### CODE REVIEW

**Review by dvsrepo — CHANGES_REQUESTED** [review]
Very cool! My only request would be to add the relevant user-facing classes and methods in the API reference:

https://github.com/recognai/rubrix/blob/master/docs/reference/python_client.rst

**Comment by frascuchon** [comment]
> Very cool! My only request would be to add the relevant user-facing classes and methods in the API reference:
> 
> https://github.com/recognai/rubrix/blob/master/docs/reference/python_client.rst

Take a look to new toc structure. Now, there is a global python section including client and metrics (and monitoring and labeling in a future)

File: docs/reference/python/python_metrics.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.. _python_metrics:
+
+Metrics (Experimental)
+======================
+
+Here we describe the available of Rubrix:
```
Comment (dvsrepo) [inline docs/reference/python/python_metrics.rst]: Here we describe the available metrics in Rubrix:

File: docs/reference/python/python_metrics.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Text classification: Metrics for text classification
+- Token classification: Metrics defined for token classification
```
Comment (dvsrepo) [inline docs/reference/python/python_metrics.rst]: - Token classification: Metrics for token classification

**Review by dvsrepo — CHANGES_REQUESTED** [review]

**Comment by dvsrepo** [comment]
> > Very cool! My only request would be to add the relevant user-facing classes and methods in the API reference:
> > https://github.com/recognai/rubrix/blob/master/docs/reference/python_client.rst
> 
> Take a look to new toc structure. Now, there is a global python section including client and metrics (and monitoring and labeling in a future)

Looks perfect! I've just suggested two edits

File: docs/reference/python/python_metrics.rst (hunk 1 above)
Comment (frascuchon) [reply docs/reference/python/python_metrics.rst]: Done!

File: docs/reference/python/python_metrics.rst (hunk 2 above)
Comment (frascuchon) [reply docs/reference/python/python_metrics.rst]: Done!

**Review by dvsrepo — APPROVED** [review]
💯 !

**Pull request merged**